File: FirstClass/Python/problem_23.py
import matplotlib.pyplot as plt
import numpy as np 
import math

if __name__ == '__main__':
    t = np.linspace (-2, 2, 1000)
    n = 2
    y = np.exp ( -0.05 * t * n) * np.cos (2 * math.pi * t)
    plt.plot (t, y)
    plt.axis ([-2, 2, -12, 12])
    plt.plot ([-2.2, 2.2], [0, 0], [0, 0], [12, -12])
    plt.show ()

File: FirstClass/problem23.m
t = -2:0.01:2;
n = 2;
y = exp (-0.05*n*t) .* cos (2 * pi * t);
plot (t, y)
axis ([-2 2 -12 12]);
hold on;
plot ([-2.2 2.2], [0 0], 'r', [0 0], [12 -12], 'r');
hold off;
grid on;

